rust/cjdns_sys/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "cjdns_sys"
+version = "0.1.0"
+authors = ["Caleb James DeLisle <cjd@cjdns.fr>"]
+edition = "2018"
+build = "build.rs"
+
+[dependencies]
+sodiumoxide = "0.2"
```

sodiumoxide has the [serde feature on by default](https://github.com/sodiumoxide/sodiumoxide#optional-features). If you don't use serde anywhere in your code, disabling the serde feature will significantly speed up compile time.
```suggestion
sodiumoxide = { version = "0.2", default-features = false, features = ["std"] }
```
This suggestion, if applicable, should be applied to all instances of sodiumoxide in a Cargo.toml